doc/releases/changelog-dev.md

```diff
@@ -181,6 +181,9 @@
   [(#4391)](https://github.com/PennyLaneAI/pennylane/pull/4391)
 
 <h3>Bug fixes 🐛</h3>
+
+* Allow sparse matrix calculation of `SProd`s containing a `Tensor`.
+  [(#4424)](https://github.com/PennyLaneAI/pennylane/pull/4424)
```

Comment:
We could potentially add a comment about recommending the use of `wire_order` over `wires` in `Tensor`, but I don't really think it matters that much.